ddlitlab2024/dataset/imports/strategies/bitbots.py

```diff
@@ -0,0 +1,282 @@
+from contextlib import contextmanager
+from datetime import datetime
+from enum import Enum
+from pathlib import Path
+
+import cv2
+import numpy as np
+from mcap.reader import make_reader
+from mcap.summary import Summary
+from mcap_ros2.decoder import DecoderFactory
+
+from ddlitlab2024.dataset import logger
+from ddlitlab2024.dataset.imports.model_importer import ImportMetadata, ImportStrategy, ModelData
+from ddlitlab2024.dataset.models import (
+    DEFAULT_IMG_SIZE,
+    GameState,
+    Image,
+    JointCommands,
+    JointStates,
+    Recording,
+    RobotState,
+    TeamColor,
+)
+from ddlitlab2024.utils.utils import camelcase_to_snakecase, shift_radian_to_positive_range
+
+DATETIME_FORMAT = "%d.%m-%Y %H:%M:%S"
+
+
+class GameStateMessage(Enum):
+    INITIAL = 0
+    READY = 1
+    SET = 2
+    PLAYING = 3
+    FINISHED = 4
+
+
+TOPIC_LIST_GO = [
+    "/DynamixelController/command",
+    "/animation",
+    "/audio/audio",
+    "/ball_obstacle_active",
+    "/ball_position_relative_filtered",
+    "/ball_relative_filtered",
+    "/ball_relative_movement",
+    "/balls_relative",
+    "/camera/camera_info",
+    "/camera/image_to_record",
+    "/cmd_vel",
+    "/debug/approach_point",
+    "/debug/ball_twist",
+    "/debug/dsd/body_behavior/dsd_current_action",
+    "/debug/dsd/body_behavior/dsd_stack",
+    "/debug/dsd/body_behavior/dsd_tree",
+    "/debug/dsd/hcm/dsd_current_action",
+    "/debug/dsd/hcm/dsd_stack",
+    "/debug/dsd/hcm/dsd_tree",
+    "/debug/dsd/localization/dsd_current_action",
+    "/debug/dsd/localization/dsd_stack",
+    "/debug/dsd/localization/dsd_tree",
+    "/debug/used_ball",
+    "/debug/which_ball_is_used",
+    "/diagnostics",
+    "/diagnostics_agg",
+    "/field/map",
+    "/gamestate",
+    "/goal_pose",
+    "/head_mode",
+    "/motion_odometry",
+    "/pose_with_covariance",
+    "/robot_state",
+    "/robots_relative",
+    "/robots_relative_filtered",
+    "/rosout",
+    "/strategy",
+    "/system_workload",
+    "/team_data",
+    "/tf",
+    "/tf_static",
+    "/time_to_ball",
+]
+
+TOPIC_LIST_RBC24 = [
+    "/DynamixelController/command",
+    "/audio/audio",
+    "/ball_position_relative_filtered",
+    "/balls_relative",
+    "/camera/camera_info",
+    "/camera/image_proc",
+    "/cmd_vel",
+    "/core/power_switch_status",
+    "/debug/dsd/body_behavior/dsd_stack",
+    "/debug/dsd/body_behavior/dsd_tree",
+    "/debug/dsd/hcm/dsd_current_action",
+    "/debug/dsd/hcm/dsd_stack",
+    "/debug/dsd/hcm/dsd_tree",
+    "/debug/dsd/localization/dsd_stack",
+    "/debug/dsd/localization/dsd_tree",
+    "/diagnostics",
+    "/diagnostics_agg",
+    "/field/map",
+    "/joint_states",
+    "/motion_odometry",
+    "/pose_with_covariance",
+    "/robot_state",
+    "/robots_relative",
+    "/robots_relative_filtered",
+    "/rosout",
+    "/strategy",
+    "/system_workload",
+    "/tf",
+    "/tf_static",
+    "/time_to_ball",
+    "/workspace_status",
+]
```

Comment:
These seem to be unused (at least in this file). What purpose do these topic-lists have?

Reply:
These were just here for documentation purposes, while figuring out the names of our topics in different rosbags. I'll remove them :+1: